test/testdata/runner/ivr_dial.json

```diff
@@ -0,0 +1,111 @@
+{
+    "flows": [
+        {
+            "uuid": "90420633-8c92-4480-940a-382cdd6a33b9",
+            "name": "IVR Redirect",
+            "spec_version": "13.0",
+            "language": "eng",
+            "type": "voice",
+            "localization": {},
+            "nodes": [
+                {
+                    "uuid": "75656148-9e8b-4611-82c0-7ff4b55fb44a",
+                    "router": {
+                        "type": "switch",
+                        "wait": {
+                            "type": "dial",
+                            "phone": "@fields.supervisor_phone"
+                        },
+                        "categories": [
+                            {
+                                "uuid": "71c901c4-b3a4-4a54-b63a-f0330e2988a9",
+                                "name": "Answered",
+                                "exit_uuid": "379c8d64-3ddf-4d37-9e54-ef29ebc1020d"
+                            },
+                            {
+                                "uuid": "50644828-da67-4602-a3ba-ab769d497040",
+                                "name": "No Answer",
+                                "exit_uuid": "131f10b1-797a-44d5-bfe7-528af7ea439c"
+                            },
+                            {
+                                "uuid": "7c837517-621b-46c6-94e0-5e03699ee67e",
+                                "name": "Busy",
+                                "exit_uuid": "52597931-6f0c-4fa3-9bdc-3c619664cb61"
+                            },
+                            {
+                                "uuid": "9f3cd249-c1be-4fe7-9503-e2bf2b988536",
+                                "name": "Failed",
+                                "exit_uuid": "fda29737-867c-45d7-89ab-f9c97a3f08cb"
+                            }
+                        ],
+                        "default_category_uuid": "9f3cd249-c1be-4fe7-9503-e2bf2b988536",
+                        "operand": "@(default(resume.dial.status, \"\"))",
```

Comment:
In the case that the wait can't make a valid tel URN, it skips itself so we fall through this router without a `@resume`, landing in the `Failed` default category. Wrapping this expression in `default()` ensures we don't generate a confusing expression error event.